apps/web/app/not-found.tsx

```diff
@@ -1,13 +1,81 @@
+"use client";
+// 3rd party
+import { Heading } from "react-aria-components";
 // repo
 import Link from "@repo/ui/link";
+import Image from "@repo/ui/image";
+import Footer from "@repo/ui/footer";
+import PreFooter from "@repo/ui/pre-footer";
+import { STR_WEBSITE_NAME } from "@repo/constants/src/constants";
+// local
+import "./not-found.css";
 
 export default function NotFound() {
-  // TODO - need a design for this
   return (
-    <div className="u-container">
-      <h2>Not Found</h2>
-      <p>Could not find requested resource</p>
-      <Link href="/">Return Home</Link>
-    </div>
+    <>
+      <section>
+        <div className="u-container web-NotFound">
+          <Heading className="web-NotFound--Title" level={1}>
+            404 - Oops, looks like we're lost
+          </Heading>
+          <Image
+            className="web-NotFound--Image"
+            src={"error-404.svg"}
+            alt=""
+            width={403}
+            height={320}
+          />
+          <p className="web-NotFound--SubTitle">
+            The page you’re looking for might have been removed, moved, or is
+            temporarily unavailable.
+          </p>
+          <p>We suggest:</p>
+          <ul>
+            <li>Checking that the web URL has been entered correctly</li>
+            <li>
+              Going to the {STR_WEBSITE_NAME} Walkthrough{" "}
+              <Link href="/">Home Page</Link> and browse through the available
```

Comment:
Did we get this text from the client or did we add it in to adjust for the correct context?

If it's not from the client, I would suggest adding "Walkthrough" into the link so that the link has more context for where it will go!

Reply:
Good idea. Updated for clarity!